src/Controls/src/Core/Handlers/Items/iOS/TemplatedCell.cs

```diff
@@ -11,8 +11,19 @@ namespace Microsoft.Maui.Controls.Handlers.Items
 {
 	public abstract class TemplatedCell : ItemsViewCell
 	{
-		public event EventHandler<EventArgs> ContentSizeChanged;
-		public event EventHandler<LayoutAttributesChangedEventArgs> LayoutAttributesChanged;
```

Comment:
So this might be a bug in:

* https://github.com/jonathanpeppers/memory-analyzers

It seems like it should have warned on these events. It *did* warn in the place we subscribed, though.

Reply:
Oh this is in `Microsoft.Maui.Controls.dll` and we don't have the analyzer here yet, it did warn:
* https://github.com/dotnet/maui/pull/18318#issuecomment-1833981761